prac_06/guitars.py

```diff
@@ -0,0 +1,15 @@
+from prac_06.guitar import Guitar
+
+guitars = []
+print("My guitars!")
+name = input("Name: ")
+while name != "":
+    year = int(input("Year: "))
+    cost = float(input("Cost: $"))
+    guitars.append(Guitar(name, year, cost))
+    print("")
```

Comment:
I think the prac sample output wanted us to display an additional line with details of the guitar added, for each guitar the user adds. To do that, I created the guitar object variable first (I called it `guitar`; the sample solution calls it `guitar_to_add`), then appended that to the list using `guitars.append(guitar)`, so you could then use `print({guitar} "added.")`

Reply:
Thanks for catching that, must have slipped my mind.